strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/ClassFileIndex.java

```diff
@@ -163,7 +163,6 @@ private void findParents(JavaClass jclass, Set<String> changedParents) {
             if (changedParents.add(parent.getName())) {
 
                 logger.log(Level.FINEST, String.format("%s test has been added because it depends on %s", parent.getName(), jclass.getName()));
```

Comment:
change it to shorter form: 
`logger.finest("%s test has been added because it depends on %s", parent.getName(), jclass.getName())`